winui/Kanban/Getting-started.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: post
+title: Getting Started with WinUI Kanban control | Syncfusion
+description: Learn here all about getting started with Syncfusion WinUI Kanban (SfKanban) control, its elements, and more.
+platform: winui
+control: SfKanban
+documentation: ug
+---
+# Getting Started with WinUI Kanban (SfKanban)
+
+This section provides an overview of working with WinUI Kanban and a step-by-step guide to configuring and using it effectively in your application.
```

Comment:
This section provides a quick overview of how to get started with the WinUI Kanban control (SfKanban) for WinUI and a walk-through to configure the WinUI Kanban control in a real-time scenario. Follow the steps below to add WinUI Kanban control to your project.

Reply:
changed content